invenio/ext/confighacks.py

```diff
@@ -77,9 +79,8 @@ def __getattr__(self, name):
             elif name == '__path__':
                 return os.path.dirname(__file__)
             try:
-                from invenio.base.helpers import utf8ifier
```

Comment:
This hack is important for *legacy* pages. We might however raise deprecation warning for Invenio 2.3.